server/models/User.js

```diff
@@ -189,6 +200,13 @@ export default (Sequelize, DataTypes) => {
           };
         },
       },
+      setterMethods: {
+        name(value) {
+          const { firstName, lastName } = splitName(value);
+          this.setDataValue('firstName', firstName);
```

Comment:
I'm not against this change (can't hurt) but just mentioning that we're aiming to remove these fields in the future

Reply:
This was breaking several tests due to `createUserWithCollective`. I rolled back.